### ISSUE TITLE
Garmin Bounce

### ISSUE THREAD

**Issue by neebski** [cyberjunky/home-assistant-garmin_connect#87]
Was curious if the Garmin Bounce would be available as a location tracker. Seems like a great device for my kids since I don't want to get them phones. If I could support location tracking in HA it would be great!

**Comment by kristianschneider** [comment]
Curious on this too. So I'm observerving

**Comment by StillClueless** [comment]
Same here. Curious if the location data can used in HA?

**Comment by cyberjunky** [comment]
Is this data visible in Garmin Connect web site, if so can you show it (anonymized)

**Comment by neebski** [comment]
Can't say for sure. I want to get a bounce for my kids but haven't yet.
Maybe someone else from the community can answer.

**Comment by FuzzyMistborn** [comment]
No, the Bounce is not visible in Connect.  It's a separate app/interface.
